packages/uui-color-picker/lib/uui-color-picker.element.ts

```diff
@@ -412,7 +415,7 @@ export class UUIColorPickerElement extends LabelMixin('label', LitElement) {
 
   private _syncValues() {
     this.inputValue = this.getFormattedValue(this.format);
-    this.value = this.inputValue;
+    this._value = this.inputValue;
```

Comment:
If you don't set `this.value` then you never call `setColor`, is that intentional?

```suggestion
    this.value = this.inputValue;
```

Reply:
Yes, because this function is being called by setColor. It would be an infinite loop.